productMods/templates/freemarker/edit/forms/addProjectUpdate.ftl

```diff
@@ -100,8 +100,8 @@ Set this flag on the input acUriReceiver where you would like this behavior to o
     <#--</p>-->
 
     <p>
```

Comment:
As in the above code where we hide the input elements for modifiedBy and modifiedOn, do we need to remove the p tags?